modules/admin_manual/pages/installation/deployment_recommendations/nfs.adoc

```diff
@@ -0,0 +1,146 @@
+= Network File System (NFS) Deployment Recommendations
+:keywords: nfs, network file system, nfsv4, mtu, async, noasync
+:description: This guide covers the official ownCloud NFS (Network File System) deployment recommendations.
+:autofs-url: https://help.ubuntu.com/community/Autofs
+:nfs-read-write-delegations-url: https://tools.ietf.org/html/rfc7530#section-1.4.6
+:mount-man-page-url: http://man7.org/linux/man-pages/man8/mount.8.html
+:lockd-url: https://docs.oracle.com/cd/E19455-01/806-0916/rfsrefer-9/index.html
+:nfs-strong-security-architecture-url: https://tools.ietf.org/html/rfc7530#section-3 
+:rpc-statd-url: https://linux.die.net/man/8/rpc.statd
+:nfs-man-page-url: https://linux.die.net/man/5/nfs
+
+ownCloud recommends using NFS for any scenario other than local storage. 
+It has solid performance and is very stable.
+This document contains ownCloud’s official deployment recommendations.
+
+There can be different scenarios where ownCloud's storage is located on an NFS mount (primary/secondary). Also in some scenarios multiple application servers can use the same NFS mount.
+
+== Use NFSv4
+
+We recommend using NFSv4 over previous versions for a number of key reasons.
+These are:
+
+* *Improved Security:* It {nfs-strong-security-architecture-url}[mandates a strong security architecture]. It does not require {rpc-statd-url}[rpc.statd] or {lockd-url}[lockd]. As a result, it only uses port 2049.
+* *Improved Reliability:* Uses TCP by default instead of UDP.
+* *Improved Performance:* It uses Multi-Component Messages, which reduce network traffic. It is capable of using a 32KB page size, compared to the default, 1024 bytes.
+* Use of {nfs-read-write-delegations-url}[Read/Write Delegations].
+
+== Tune the Mount Parameters
+
+=== _netdev
+
+Use this option to ensure that the NFS storage is available before ownCloud starts.
```

Comment:
Better: `Use this option to ensure that the network is up and running before NFS tries to connect.`
ownCloud does not start. It is reactive...
Maybe you want to write: `File and database services need to be up and running before accessing ownCloud.`

Reply:
I just re-read the setting definition, and I see that you're correct, @mmattel.